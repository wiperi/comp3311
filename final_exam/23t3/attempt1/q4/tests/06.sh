python3 q4.py "Jessica Ou Yang"
