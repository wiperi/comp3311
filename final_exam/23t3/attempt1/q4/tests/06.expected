t1=44, t2=44
Not improving
